database.sql

```diff
@@ -24,9 +24,11 @@ DROP TABLE IF EXISTS `level`;
 /*!50503 SET character_set_client = utf8mb4 */;
 CREATE TABLE `level` (
   `id` int NOT NULL AUTO_INCREMENT,
-  `name` varchar(30) DEFAULT NULL,
+  `name` varchar(30) NOT NULL,
   `description` text,
-  `content` varchar(2000) DEFAULT NULL,
+  `content` longtext NOT NULL,
+  `width` int NOT NULL,
+  `height` int NOT NULL,
```

Comment:
if you're working on the database, it's time to get rid of the "01" and use a cell entity

Reply:
done, though it'll be more annoying to review with everything done at once ;)